.eslintrc.js

```diff
@@ -164,6 +164,13 @@ module.exports = {
         "react/display-name": "warn",
         "react/prop-types": "warn"
       }
-    }
+    },
+    {
+      // Remove limits on number of lines in test files
+      "files": ["__tests__/**"],
+      "rules": {
+        "max-lines": "off"
```

Comment:
Any particular reason you decided to turn of the `max-lines` instead of upping the limit to say 500 lines?